src/sentry/migrations/0470_convert_unresolved_to_set_escalating_activitytype.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-05-26 07:32
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.types.activity import ActivityType
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def convert_to_set_escalating(apps, schema_editor):
+    Activity = apps.get_model("sentry", "Activity")
+
+    for activity in RangeQuerySetWrapperWithProgressBar(
+        Activity.objects.filter(type=ActivityType.SET_UNRESOLVED)
+    ):
+
+        if not activity.data.get("forecast"):
+            continue
+
+        activity.update(type=ActivityType.SET_ESCALATING)
```

Comment:
```suggestion
        activity.type = ActivityType.SET_ESCALATING
        activity.save(update_fields=["type"])
```

There is no `update()` method on the django historical models. You need to use `save()` instead.